plugins/inputs/fireboard/fireboard.go

```diff
@@ -0,0 +1,126 @@
+package fireboard
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+// Fireboard gathers statistics from the fireboard.io servers
+type Fireboard struct {
+	AuthToken string
+	URL       string
```

Comment:
Adding struct tags would be good, as it enforces the correct case and sometimes avoids parsing abiguity (there is none here but they are nice for consistency):
```go
AuthToken string `toml:"auth_token"`
URL       string `toml:"url"`
```